eessi_bot_job_manager.py

```diff
@@ -356,8 +393,9 @@ def process_new_job(self, new_job):
             if "comment_id" in new_job:
                 new_job_comments_cfg = config.read_config()[config.SECTION_NEW_JOB_COMMENTS]
                 dt = datetime.now(timezone.utc)
-                update = "\n|%s|released|" % dt.strftime("%b %d %X %Z %Y")
-                update += f"{new_job_comments_cfg[config.NEW_JOB_COMMENTS_SETTING_AWAITS_LAUNCH]}|"
+                update = "\n|%s|%s|" % (dt.strftime("%b %d %X %Z %Y"), job_status)
+                description_col_fmt = new_job_comments_cfg[config.NEW_JOB_COMMENTS_SETTING_AWAITS_LAUNCH]
```

Comment:
I guess a minor detail is that by default, this now still says:

```
job id 9873473 awaits release by job manager
```

Even if the `delayed_begin` protocol is configured. Then again, people can customize this message in their config if they really wanted to. So... no big deal. I say we leave it for now, but if you want you can make a tiny follow-up PR and I'll merge it

Reply:
Looked into doing that programmatically defining two template strings then use them depending on the job handover protocol. However, because we set the slurm parameters early (and thus also the value for delaying the job with `--begin=now+delay`), making this "nice" needs more changes. A simple change could be to adjust the string to, for example, `... awaits release or is eligible to start soon` or something similar.

If we would want to make it "nice", we somehow would have to store the delay and then could write something like `... awaits release by job manager` or `... is eligible to run in 'delay' seconds`.

Reply:
Never mind, I think I found an acceptable way to implement this in https://github.com/EESSI/eessi-bot-software-layer/pull/297/commits/dc5c2892acf9be7ad8e0f8a89c14921e55249690

Reply:
Nice! There seem to be some failing unit tests though, I'm not entirely sure why... you probably know better :)